api/v1beta1/types.go

```diff
@@ -0,0 +1,196 @@
+/*
```

Comment:
What is the filename semantics? `types` vs `gslb_types`?

Reply:
@ytsarev  good question, 
It lives originally in `k8s.io/api/extensions/v1beta1/types.go`.

Reply:
`upstream_types.go` maybe?

Reply:
Renamed, was just following original convention `upstream_types.go` is well descriptive.